src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php

```diff
@@ -53,6 +54,10 @@
      */
     public function __construct(PropertyNameCollectionFactoryInterface $propertyNameCollectionFactory, PropertyMetadataFactoryInterface $propertyMetadataFactory, ResourceMetadataFactoryInterface $resourceMetadataFactory, int $maxJoins = 30, bool $forceEager = true, RequestStack $requestStack = null, SerializerContextBuilderInterface $serializerContextBuilder = null, bool $fetchPartial = false, ClassMetadataFactoryInterface $classMetadataFactory = null)
     {
+        //if (null !== $this->serializerContextBuilder) {
+        //    @trigger_error('Passing an instance of "%s" is deprecated since version 2.2 and will be removed in 3.0. Use the "normalization_context" of the data provider\'s context instead.', E_USER_DEPRECATED);
+        //}
```

Comment:
You want to keep this dead code?

Reply:
No I'll drop it, I try to debug something.